modules/avcapture/avcapture.m

```diff
@@ -150,8 +150,22 @@ + (AVCaptureDevicePosition)get_position:(const char *)name
 + (AVCaptureDevice *)get_device:(AVCaptureDevicePosition)pos
 {
 	AVCaptureDevice *dev;
-
+#if (__IPHONE_OS_VERSION_MIN_REQUIRED >= 100000 || \
+     __MAC_OS_X_VERSION_MIN_REQUIRED >= 101500)
+	AVCaptureDeviceDiscoverySession *discoverySession;
+
+	discoverySession = [
+		AVCaptureDeviceDiscoverySession
+		discoverySessionWithDeviceTypes:
+			@[AVCaptureDeviceTypeBuiltInWideAngleCamera,
+			  AVCaptureDeviceTypeExternalUnknown]
+		mediaType:AVMediaTypeVideo
+		position:pos
```

Comment:
I dont know how to test `position` on macOS. Maybe someone can check this on iOS.